docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -277,6 +271,58 @@ ECLCC_PATH
               </tgroup>
             </informaltable></para>
         </sect3>
+
+        <sect3 id="CT-CLI-GIT-Support">
+          <title>Git Support</title>
+
+          <para>Starting with the HPCC Systems version 8.4, the
+          <emphasis>--main</emphasis> option has been extended to support
+          compiling a query directly from a git repository. When invoked, It
+          retrieves the ECL code specified from the git repository, compiles
+          the code and runs the query (on Thor). Only the checkout will be
```

Comment:
is this only valid on Thor?

Reply:
I think so, @ghalliday can you please confirm?

Reply:
No, it is valid on any target.